tests/test_ctl.py

```diff
@@ -0,0 +1,181 @@
+import shlex
+import stat
+import subprocess
+
+import pytest
+
+from pgtoolkit import ctl, hba
+
+
+def test__args_to_opts():
+    opts = ctl._args_to_opts(
+        {
+            "encoding": "latin1",
+            "auth_local": "trust",
+            "show": True,
+            "n": True,
+            "L": "DIR",
+        }
+    )
+    assert opts == [
+        "-L DIR",
+        "--auth-local=trust",
+        "--encoding=latin1",
+        "-n",
+        "--show",
+    ]
+
+
+def test__wait_args_to_opts():
+    assert ctl._wait_args_to_opts(False) == ["--no-wait"]
+    assert ctl._wait_args_to_opts(True) == ["--wait"]
+    assert ctl._wait_args_to_opts(42) == ["--wait", "--timeout=42"]
+
+
+@pytest.fixture
+def fake_pgctl(tmp_path):
+    (tmp_path / "pg_ctl").touch()
+    c = ctl.PGCtl(tmp_path)
+
+    def run_command(args, **kwargs):
```

Comment:
Can't `kwargs` be removed here?

Reply:
No; we need to keep a compatible signature because it's called with keyword arguments in the API we are testing.